### ISSUE TITLE
install ghostbsd KDE in vbox

### ISSUE THREAD

**Issue by fabrice14** [ghostbsd/ghostbsd-build#49]
hello,
After installed ghostbsd kde 2020-04-21 in a virtualbox, It is not possible to startx or startplasma too.
startx send an error:
ld-elf.so.1: Shared object "libXau.so.6" not found, required by "xauth"

It was the same issue for the previous build.

Anyway, kde started in a vbox without installation.

**Comment by vimanuelt** [comment]
There is no logic in https://github.com/ghostbsd/ghostbsd-build/blob/master/xorg/bin/xconfig to support KDE desktop. Will need to add support.

**Comment by vimanuelt** [comment]
Added  case for kde, mate, xfce, and cinnamon.

**Comment by vimanuelt** [comment]
Feature was added to latest development builds. Please confirm whether changes correct issue.

**Comment by fabrice14** [comment]
Hello,
I found the same issue with ghostbsd kde 2020-04-25. maybe the code has not been added in this development build yet?

**Comment by fabrice14** [comment]
would you like me to test ghostBSD latest?

**Comment by vimanuelt** [comment]
No. Thanks for confirming the status.

**Comment by vimanuelt** [comment]
There is an xconfig refactoring that is in progress and needs to be completed before I can proceed with troubleshooting.

**Comment by fabrice14** [comment]
yes, I understand. this is not a big problem anyway. Thank you.

**Comment by fabrice14** [comment]
are you responsible of the image of kde in ghostbsd?

**Comment by vimanuelt** [comment]
The KDE build is a team effort. It will take some time to sort out the issue.

**Comment by vimanuelt** [comment]
Eric and I have refactored virtualbox support. You can download the latest ISO file at http://download.us.ghostbsd.org/development/amd64/latest/GhostBSD-2020-04-30-KDE.iso to test. Please send us your feedback. Thank you for your reply.

**Comment by fabrice14** [comment]
Hello,
Thank you for your reply.
I can not installed the lastest kde iso in the vbox, because the installation is failed. So I am not able to test the problem.
I am going to check that with the next build.

**Comment by fabrice14** [comment]
there is an error on zpool, image in attach
![Capture du 2020-05-01 18-54-07](https://user-images.githubusercontent.com/40697283/80823818-914d0a80-8bdd-11ea-84c1-5719af7069ff.png)

**Comment by vimanuelt** [comment]
Thanks for the feedback. 
The original issue has been solved, so I'll close this ticket. Please open another ticket regarding the ZFS pool issue. Thanks.
